### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPF ControlsAndLayout_Controls TabControl_Keyboard: Tab get selected when keyboard focus moves from one tab to other.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1468]
Copied fro,m VSO [956741](https://devdiv.visualstudio.com/DevDiv/_queries/edit/956741/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 WPF

.NET Core SDK version:  3.0.100-preview8-013442

Windows Version: Windows10
ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.

2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

Repro Steps:

1) Open Visual Studio.

2) Hit on File and select Open->Project/Solution.

3) Run exe file for "ControlsAndLayout" from the build app folder provided. 
4)Main Window for "ControlsAndLayout" solution will appear. 
5)From the Sample Library pane under Controls, select "TabControl". 
6) Navigate to any menu item using keyboard.

Actual:

Tab get selected when keyboard focus moves from one tab to other.
Image

Expected:
Only On focus any item should not get selected.

User Impact:

Keyboard only user will get confused while navigating as it is directly get selected.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 3.2.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

**Comment by rladuca** [comment]
![image](https://user-images.githubusercontent.com/20006987/64462573-c1c3bf80-d0b5-11e9-9780-976ea54d249b.png)

A tab control is really a collection of underlying sibling controls (as is shown by the automation tree above).  When you tab across, you are selection different controls.  The content shown is associated with the specific tab item you have selected.  There is no accessibility violation here as these are distinct controls within WPF that are grouped together.
